pycsw/ogc/api/records.py

```diff
@@ -1295,6 +1295,13 @@ def record2json(record, url, collection, mode='ogcapi-records'):
         else:
             record_dict['time'] = record.time_begin
 
+    if mode == 'stac-api':
+        if record.time_begin is not None or record.time_end is not None:
```

Comment:
I think we need `and` here, the spec says:
`So if you use start_datetime you need to add end_datetime and vice-versa. Both fields are also REQUIRED if the datetime field is set to null. The datetime property in a STAC Item and these fields are not mutually exclusive.`
If only one exists, we just set `datetime` from it.

Reply:
https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#date-and-time-range